test/integration/acceptance/revoke_permission_test.cpp

```diff
@@ -355,7 +355,10 @@ namespace grantables {
 
                        ASSERT_NE(message.find("did not pass verification"),
                                  std::string::npos)
-                           << "Fail reason: " << message;
+                           << "Fail reason: " << message
+                           << "\nRaw status:" << status.toString();
+                       // we saw empty message was received once
```

Comment:
maybe move this line above? since comment usually goes earlier the things they're document